### ISSUE TITLE
How to build it and use it?

### ISSUE THREAD

**Issue by bagusflyer** [ReactiveCocoa/ReactiveViewModel#41]
Can you add document about how to build it and how to use it?

**Comment by joshaber** [comment]
The [README](https://github.com/ReactiveCocoa/ReactiveViewModel#getting-started) contains some information about building.

**Comment by bagusflyer** [comment]
Don't know how it works.

**Comment by joshaber** [comment]
Are there specific questions you have? Why do you want to use it in the first place?

**Comment by bagusflyer** [comment]
Sorry. I didn’t say it clearly. I’m just learning MVVM, ReactiveCocoa and found this library and want to have a try. After downloading the git file and unzip it, I found ReactiveCocoa is missing. (Normally I’m using Podfile for ReactiveCocoa). I don’t know what to do next.

**Comment by joshaber** [comment]
True, it does not include ReactiveCocoa in the repository itself. ReactiveCocoa is a dependency included as a submodule, so you'd need to clone the repository and clone the submodules recursively to get it. Or use Carthage or CocoaPods to pull in the dependencies for you.

**Comment by alper** [comment]
There's unfortunately no Podfile in the repository to be able to quickly install dependencies.
